Essentials/src/main/java/com/earth2me/essentials/commands/Commandme.java

```diff
@@ -40,12 +40,13 @@ public void run(final Server server, final User user, final String commandLabel,
         message = FormatUtil.formatMessage(user, "essentials.chat", message);
 
         user.setDisplayNick();
-        final int radius = ess.getSettings().getChatRadius();
+        long radius = ess.getSettings().getChatRadius();
```

Comment:
```suggestion
        final long radius = ess.getSettings().getChatRadius();
```
codestyle sadge

Reply:
I change this value on line 49, so I can't make it final here.

Reply:
misread, ignore